pkg/metrics/metrics.go

```diff
@@ -234,12 +236,13 @@ func EmitSubMetric(sub *olmv1alpha1.Subscription) {
 	if sub.Spec == nil {
 		return
 	}
-	SubscriptionSyncCount.WithLabelValues(sub.GetName(), sub.Status.InstalledCSV, sub.Spec.Channel, sub.Spec.Package).Inc()
+	SubscriptionSyncCount.WithLabelValues(sub.GetName(), sub.Status.InstalledCSV, sub.Spec.Channel, sub.Spec.Package, string(sub.Spec.InstallPlanApproval)).Inc()
 	if _, present := subscriptionSyncCounters[sub.GetName()]; !present {
 		subscriptionSyncCounters[sub.GetName()] = subscriptionSyncLabelValues{
 			installedCSV: sub.Status.InstalledCSV,
 			pkg:          sub.Spec.Package,
 			channel:      sub.Spec.Channel,
+			planApproval: string(sub.Spec.InstallPlanApproval),
```

Comment:
Isn't `approvalStrategy` a better name for this variable?

Reply:
I agree, i will change that.